lib/widgets/expandable_fab.dart

```diff
@@ -0,0 +1,140 @@
+import 'package:flutter/material.dart';
+
+class ExpandableFab extends StatefulWidget {
+  const ExpandableFab({Key? key, required this.items}) : super(key: key);
+
+  final List<ExpandableItem> items;
+
+  @override
+  _ExpandableFabState createState() => _ExpandableFabState();
+}
+
+class _ExpandableFabState extends State<ExpandableFab>
+    with SingleTickerProviderStateMixin {
+  late final AnimationController _controller;
+  late final Animation<double> _animateIcon;
+  late final Animation<double> _expandAnimation;
+  late bool _open;
+
+  @override
+  void initState() {
+    _open = false;
+    _controller = AnimationController(
+      value: _open ? 1.0 : 0.0,
```

Comment:
chore: `_open` will always be false inside of `initState` so this should be `value: 0,`.